iree/tools/init_passes.h

```diff
@@ -148,10 +148,6 @@ inline void registerMlirPasses() {
   createConvertLinalgToAffineLoopsPass();
   createConvertLinalgToLLVMPass();
 
-  // Loop
-  createParallelLoopFusionPass();
-  createParallelLoopTilingPass();
```

Comment:
Are those no longer used in IREE? Also LoopOps was renamed to SCF, these passes still exist ([include/mlir/Dialect/SCF/Passes.h](https://github.com/llvm/llvm-project/blob/master/mlir/include/mlir/Dialect/SCF/Passes.h)).

Reply:
Should be corrected now.